web-client/src/presenter/actions/setDefaultTableSortAction.test.js

```diff
@@ -65,41 +66,40 @@ describe('setDefaultTableSortAction', () => {
       },
     });
 
-    expect(state.tableSort.sortField).toEqual('completedAt');
+    expect(state.tableSort.sortField).toEqual('createdAt');
     expect(state.tableSort.sortOrder).toEqual(DESCENDING);
   });
 
-  it('should not set the tableSort if role is not ADC', async () => {
+  it('the completed should be sorted by createdAt DESCENDING for adc user', async () => {
     const { state } = await runAction(setDefaultTableSortAction, {
       modules: {
         presenter,
       },
       props: {
-        box: 'inbox',
-        queue: 'section',
+        box: 'completed',
       },
       state: {
         user: {
-          role: 'docketclerk',
+          role: 'adc',
         },
       },
     });
 
-    expect(state.tableSort).toBeUndefined();
+    expect(state.tableSort.sortField).toEqual('completedAt');
+    expect(state.tableSort.sortOrder).toEqual(DESCENDING);
   });
 
-  it('should not set the tableSort if queue is not section', async () => {
+  it('should not set the tableSort if role is not ADC', async () => {
     const { state } = await runAction(setDefaultTableSortAction, {
       modules: {
         presenter,
       },
       props: {
         box: 'inbox',
-        queue: 'my',
       },
       state: {
         user: {
-          role: 'adc',
+          role: 'docketclerk',
```

Comment:
Could you explicitly set the tableSort here in the `arrange` portion of the test so we can be sure this will fail? Thanks!

Reply:
Good catch, added the default sort order